src/Type/Schema.php

```diff
@@ -51,38 +51,35 @@
  */
 class Schema
 {
-    /** @var SchemaConfig */
-    private $config;
+    private SchemaConfig $config;
 
     /**
      * Contains currently resolved schema types
      *
-     * @var Type[]
+     * @var array<Type>
      */
-    private $resolvedTypes = [];
+    private array $resolvedTypes = [];
 
     /**
      * Lazily initialised.
      *
      * @var array<string, InterfaceImplementations>
      */
-    private $implementationsMap;
+    private array $implementationsMap;
 
     /**
-     * True when $resolvedTypes contain all possible schema types
-     *
-     * @var bool
+     * True when $resolvedTypes contains all possible schema types.
      */
-    private $fullyLoaded = false;
+    private bool $fullyLoaded = false;
 
-    /** @var Error[] */
-    private $validationErrors;
+    /** @var array<Error> */
```

Comment:
```suggestion
    /** @var array<int, InvariantViolation|Error> */
```

I took the type from the method below

Reply:
I double checked and found it is always `array<int, Error>`.